MovieDB/Scenes/Reviews/ReviewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ReviewModel.swift
+//  MovieDB
+//
+//  Created by Nguyen Hung on 6/8/19.
+//  Copyright © 2019 nguyen.van.hungd. All rights reserved.
+//
+
+struct ReviewModel {
```

Comment:
thường cái này gọi là ViewModel

```suggestion
struct ReviewViewModel {
```